app/src/main/java/com/github/libretube/ui/fragments/HomeFragment.kt

```diff
@@ -64,23 +65,23 @@ class HomeFragment : Fragment() {
         }
 
         binding.featuredTV.setOnClickListener {
-            findNavController().navigate(R.id.subscriptionsFragment)
+            findNavController().navigate(R.id.action_homeFragment_to_libraryFragment)
```

Comment:
This one should actually be `R.id.action_homeFragment_to_subscriptionsFragment` (which doesn't yet exist) to preserve the previous navigation behavior.

Reply:
Fixed